engine/swarm/configs.md

```diff
@@ -166,11 +166,18 @@ real-world example, continue to
 4.  Get the ID of the `redis` service task container using `docker ps`, so that
     you can use `docker container exec` to connect to the container and read the contents
     of the config data file, which defaults to being readable by all and has the
-    same name as the name of the config. The first command below illustrates
-    how to find the container ID, and the second and third commands use shell
+    same name as the name of the config. The first command retrieves the node ID
+    the container is running on, from the node  `ip-172-31-46-109` run the second command below illustrates
+    how to find the container ID, and the third and fourth commands use shell
     completion to do this automatically.
 
     ```console
+    $ docker service ps --format '{{ "{{" }} .Node }}' redis
```

Comment:
is this format value correct? trying to parse it but my brain can't handle

cc @thaJeztah

Reply:
OH, I see now. This needs the special `{% raw %}` and `{% endraw %}` tags before and after the code block, to escape the double curly braces.

Reply:
So, the example was written with a "test" environment (single node) in mind. Finding the right node where the task landed up on would be useful for multi-node swarm clusters.

That said, "from now on command should be run on that node" may also not be correct, as `docker service` commands can only be run from a manager node, so if the task landed on a worker node, those commands won't work.

Given that this step was only to "verify that the config was applied", I'm wondering if we could come up with a better way to show that (without requiring direct access to the node). e.g. could be a "web" service (nginx container, e.g.) that shows the config, or a service that logs the contents of the config (which can be viewed using `docker service logs`).

Reply:
Thanks for the comments 💯 

What I meant with "from now on command should be run on that node" is that the rest of the commands in `Number 4` aside from the first command (the comment came after the first command) should be executed in the worker node.